app/views/partners/_partner_row.html.erb

```diff
@@ -4,18 +4,20 @@
   <td><%= link_to partner_row.email, "mailto:#{partner_row.email}" %></td>
   <td>
     <% case status %>
-      <% when "pending" %>
+    <% when "uninvited" %>
+      <button class="btn btn-xs bg-teal">Uninvited</button>
+      <% when "invited" %>
         <%= link_to approve_partner_partner_path(partner_row), class: "btn btn-xs bg-maroon" do %><%= status.humanize %><% end %>
       <% when "awaiting_review" %>
         <%= link_to approve_partner_partner_path(partner_row), class: "btn btn-xs bg-orange" do %><%= status.humanize %><% end %>
       <% when "approved" %>
         <%= link_to approve_partner_partner_path(partner_row), class: "btn btn-xs bg-olive" do %><%= status.humanize %><% end %>
```

Comment:
I think I missed this one when it was added -- could you change these to use the appropriate helpers from `UiHelper`? You should be able to pass a color option to them -- I suspect they'd all be `view_button_to` though.

Reply:
This is fine. Just as an FYI, @Nirvikalpa108 and I are going to propose some changes to this table here soon. We can cope with these changes if the land before ours.